packages/sdk/src/react/hooks/useRoyaltyPercentage.tsx

```
@@ -39,9 +39,8 @@ const fetchRoyaletyPercentage = async (
 		]);
 
 		return royaltyPercentage;
-	} catch {
-		//TODO: dont swallow errors
-		return 0n;
```

On tests, it always returned `0n`, which `dont swallow errors` TODO makes more sense. I made it throw error in this case.